modules/libcom/src/error/errlog.c

```diff
@@ -476,8 +476,7 @@ int errlogSetConsole(FILE *stream)
 {
     errlogInit(0);
     epicsMutexMustLock(pvt.msgQueueLock);
-    pvt.console = stream ? stream : stderr;
-    pvt.ttyConsole = isATTY(pvt.console);
+    pvt.console = stream;
```

Comment:
What happens with `errlogSetConsole(stdout)` or `errlogSetConsole(stderr)`?

I think an explicit test for the three special `std*` is necessary, with the result stored alongside `pvt.console=NULL`.  (passing `stdin` should be an error)

Also, please do not remove `pvt.ttyConsole`.  `isATTY()` is a relatively expensive function, so I think caching the result makes sense.  (or look at speeding up `isATTY()`)